CMSgov/mint-app
# [EASI-4025] FE work for the Daily Digest Notification

# EASI-3949, EASI-4025


## Changes and Description

- Add `DailyDigestCompleteActivityMeta` fragment to `GetNotifications.ts` graphql file
- Add new translations for the Daily Digest to `notifications.ts` translation file
- Created `DailyDigest` component and its test
- Add `DailyDigest` component to `IndividualNotifications.tsx`
- Moved a bunch of util functions to a utils file
- 



## How to test this change

1. seed the app
2. navigate to notifications
3. See that a new daily digest is there
4. click `View digest` and verify that the "unread" state is gone
5. ensure the daily digest matches the figma




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
